01-introduction.Rmd

```diff
@@ -113,121 +167,140 @@ Because R is open source, these developments can easily build on previous work,
-An obsolete S3 class called "Map" was used to contain a geometry nested list and a data.frame, but work on the class representation in **maptools** fed directly into **sp** before **sp** was published on CRAN.
+A package not mentioned in these overviews was **maptools** written by Nicholas Lewin-Koh [@bivand_maptools:_2017]. 
+Initially, it just contained a wrapper around [shapelib](http://shapelib.maptools.org/), and permitted the reading of ESRI Shapefiles into geometry nested lists. 
+The corresponding and nowadays obsolete S3 class called "Map" stored this list alongside an attribute dataframe. 
+The work on the "Map" class representation was nevertheless important since it directly fed into **sp** prior to its publication on CRAN.
 
-Two years later an extended review of spatial packages was published [@hornik_approaches_2003]. 
-Around this time the development of R's spatial capabilities started to be augmented with links to external libraries, especially GDAL) and PROJ.4, which facilitate geographic data I/O (covered in chapter \@ref(read-write)) CRS transformations respectively.
-Based on the wide range of data formats that could be read-in by GDAL, @hornik_approaches_2003 also proposed a spatial data class system, including support for points, lines, polygons and grids.
-These ideas can be seen in **rgdal** and **sp**, which became foundational packages for spatial data analysis with R [@bivand_applied_2013].
+Two years later, @hornik_approaches_2003 published an extended review of spatial packages. 
+Around this time the development of R's spatial capabilities increasinlgy supported interfaces to external libraries, especially to GDAL and PROJ.4.
+These interfaces facilitated geographic data I/O (covered in chapter \@ref(read-write)) and CRS transformations, respectively.
+@hornik_approaches_2003 proposed a spatial data class system, including support for points, lines, polygons and grids based on GDAL's support for a wide range of spatial data formats.
+All these ideas contributed to the packages **rgdal** and **sp**, which became the foundational packages for spatial data analysis with R [@bivand_applied_2013].
 
-**rgdal**, first released in 2003, greatly extended R's spatial capabilities in terms of access to spatial data formats previously unavailable to R users. Initially, only raster drivers were supported, based on Tim Keitt's GDAL bindings for R.
-Importantly, it enabled storing information about coordinate reference system and allowed for map projection and datum transformation. 
-Many of these additional capabilities were developed by Barry Rowlingson and folded into the **rgdal** codebase because the same underlying external dependencies were needed.
+Naturally, **rgdal**, first released in 2003, greatly extended R's spatial capabilities in terms of access to previously unavailable spatial data formats.
+Initially, only raster drivers were supported, based on Tim Keitt's GDAL bindings for R.
```

Comment:
Since I've been using R I think - this has always worked for me:

```
cities <- readOGR(system.file("vectors", package = "rgdal")[1], "cities")
OGR data source with driver: ESRI Shapefile 
Source: "D:/R/Rlibs/rgdal/vectors", layer: "cities"
with 606 features
It has 4 fields
Integer64 fields read as strings:  POPULATION 
```